Pre-/Post-Events and Trains - final comment period (#84)
RustFestEU/blog.rustfest.eu: _posts/2019-08-11-pre-post-events-sustainable-train-travels.md

---
layout: post

> title: "Pre-/Post Events: sustainable 🚄 train travels from and to RustFest Barcelona"

Reviewer: Maybe "Pre-/Post-Conf Events" sounds better? Or just "Travel Events"?

Author: ack

> title: "Pre-/Post Events: sustainable 🚄 train travels from and to RustFest Barcelona"

Reviewer 2: ```suggestion
title: "Pre-/Post-Conf arrangements for sustainable travel (hint: 🚄)"
```

authors:
  - dns2utf8
socialImageSrc: "/assets/posts/barcelona/train-map.png"

> ---

Reviewer: We should link to both the CfP and ticket sales in each blog post.

Author: Done

So what do we need for that?

1. Is there a path to go by train?
2. How far can you go in one day?
3. Which stops are interesting?
4. Where can I meet other people?

1 and 2 you have to answer for yourself.

3 and 4 we can help. We are [collecting all pre-/post events](https://barcelona.rustfest.eu/pre_post_events/) on our website.

Beeing able to talk to your peers directly makes planning easy.
The [Rust User Forum](https://users.rust-lang.org/c/community) is a great place to organize.

### Tickets or Interrail

Usually people buy tickets from A to B, sometimes by locking in on a single connection.
However, a travel of this length has some challenges.

When traveling huge distances over many countries sometimes delays happen or a train might break down.
With a regular ticket you have to find staff and organize a reroute on the spot.

This is where Interrail saves the day.
With the Interrail global pass you buy travel days over a period of time.
So you can simply catch the next train and continue your journey.

If you buy 7 travel days within one month, you have the flexibility to ride any train outside of your home country on 7 days.

> So you have the flexibility to ride any train outside of your home country on 7 days.

Reviewer: That's a repeat of the previous sentence.

### From France or Portugal

From most bigger cities of France you can reach Barcelona within 7 to 9 hours.

* From Paris, there is a direct TGV connection 3 times a day.
* From Rennes, take the TGV via Nimes.
* From Bordeaux, with InterCity and TGV via Narbonne.

* From Lisboa Estacion Oriente there is a train to Madrid Chamartin. Since it is not a high speed train it takes 11 hours and you will probably need to continue on the second day.

Note that there is a [cheaper Interrail country pass](https://www.interrail.eu/en/interrail-passes/one-country-pass/spain-passes) and you probably only need 3 days within 1 month.

### From the North

These events are timed so that people traveling towards Barcelona can stop by.
What does that mean, let's go through a possible option:

{:.layout-auto}
|              City               |    Day    | Departure | Arrival |                                           Notes                                           |
| ------------------------------- | --------- | --------- | ------- | ----------------------------------------------------------------------------------------- |

> | Gothenburg, SE                  | Wednesday | 08:55     | 12:28   | Swing by the office before and grab a laptop and a coffee to go.                          |

Reviewer: I still find the "by the office" thing confusing. Make it "your office", "favorite coffee shop" or something, making it clear we don't have any on-site event planned.

Author: fixed

The NightJet 471 from Berlin rolls directly to Zürich.

### From the East

If you are traveling from the East it might be possible to cross routes with folks from the North and join their travel.
Lucky us there is a direct connection from Budapest to Zürich.
Let's look at it:

{:.layout-auto}
|        City         |    Day    | Departure | Arrival |                                          Notes                                          |
| ------------------- | --------- | --------- | ------- | --------------------------------------------------------------------------------------- |
| Budapest-Keleti, HU | Wednesday | 20:40     | 08:20   | Jump on the NightJet and relax.                                                         |
| Vienna, AT          | Wednesday | 23:25     | 08:20   | Same train, has multiple stops in Austria.                                              |
| Zürich HB, CH       | Thursday  |           | 08:20   | Time to walk a little in Zürich, meet people and maybe hit a shower and day of hacking. |

Note that you can take a direct train from Warsaw or Prague to Vienna.

### From the South

> We will suggest two possible options from Italy. Italy basically has one "train backbone" that goes from South to North so you basically choose how many stops you want to do. For more options and ideas visit [https://www.interrail.eu](https://www.interrail.eu).

Reviewer: ```suggestion
We suggest two possible options from Italy. Italy basically has one "train backbone" that goes from South to North so you choose how many stops you want to do. For more options and ideas visit [https://www.interrail.eu](https://www.interrail.eu).
```

{:.layout-auto}
|     City      |    Day    | Departure | Arrival |                                                       Notes                                                        |
| ------------- | --------- | --------- | ------- | ------------------------------------------------------------------------------------------------------------------ |
| Rome, IT      | Wednesday | 11:30     | 14:30   | Get a quick "FrecciaRossa" train to Milan.                                                                         |
| Milan, IT     | Wednesday | 15:05     | 22:30   | Get the "EuroCity" train to Marseille, France.                                                                     |
| Marseille, FR | Thursday  | 08:00     | 12:30   | You're in Marseille the whole night. Either sleep or meet witht friends for a night of hacking or an afterhour :-) |
| Barcelona, SP | Friday    |           | 12:30   | You're in Barcelona and you can rest a bit, visit the city and the following day you'll be cool for the Rustfets   |

> **Second option**, with one more stop and less idle time:

Reviewer: If you highlight this as is, you also need to do the same for the **First option**

We want to encourage more people to join us or plan similar travel events.

So what do we need for that?

1. Is there a path to go by train?
2. How far can you go in one day?
3. Which stops are interesting?
4. Where can I meet other people?

1 and 2 you have to answer for yourself.

3 and 4 we can help. We are [collecting all pre-/post events](https://barcelona.rustfest.eu/pre_post_events/) on our website.

Beeing able to talk to your peers directly makes planning easy.
The [Rust User Forum](https://users.rust-lang.org/c/community) is a great place to organize.

### Tickets or Interrail

Usually people buy tickets from A to B, sometimes by locking in on a single connection.
However, a travel of this length has some challenges.

When traveling huge distances over many countries sometimes delays happen or a train might break down.
With a regular ticket you have to find staff and organize a reroute on the spot.

This is where Interrail saves the day.
With the Interrail global pass you buy travel days over a period of time.
So you can simply catch the next train and continue your journey.

> If you buy 7 travel days within one month, you have the flexibility to ride any train outside of your home country on 7 days.

Reviewer 2: ```suggestion
If you buy the "7 travel days within one month" ticket, you'll have the flexibility to take any train outside of your home country for 7 days.
Included in this ticket, you're allowed <strong>one outbound journey</strong> from your country and <strong>one inbound journey</strong> to your country. You can find more information [here](https://www.interrail.eu/en/interrail-passes/what-is-interrail/how-use-your-interrail-pass) on how an Interrail train ticket work.

```

Reviewer 2: @dns2utf8 see if you like how I have reworded

Author: Will refine it a bit more

> So you have the flexibility to ride any train outside of your home country on 7 days.

Reviewer 2: ```suggestion
```
